package.json

```diff
@@ -11,6 +12,8 @@
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
     "axios": "^0.21.0",
+    "create-react-app": "^4.0.0",
+    "global": "^4.4.0",
```

Comment:
Why do we have to import both of these?

Reply:
I'm guessing we don't need both.  I'm not sure how both ended up getting imported.